app/src/main/java/org/schabi/newpipe/player/VideoPlayer.java

```diff
@@ -576,12 +580,36 @@ public void onPrepared(boolean playWhenReady) {
 
         super.onPrepared(playWhenReady);
 
+        tryMarkSponsorTimes();
+
         if (simpleExoPlayer.getCurrentPosition() != 0 && !isControlsVisible()) {
             controlsVisibilityHandler.removeCallbacksAndMessages(null);
             controlsVisibilityHandler.postDelayed(this::showControlsThenHide, DEFAULT_CONTROLS_DURATION);
         }
     }
 
+    private void tryMarkSponsorTimes() {
```

Comment:
Please just call the function `markSponsorTimes` (without the try in front of it).

Reply:
done